src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.html

```diff
@@ -49,19 +49,19 @@ <h2 class="card-pf-title">
                         <a href='https://appdev.openshift.io/docs/contrib-guide.html#_documentation' target="_blank">library</a>
                         and help us expand these offerings.
                       </ng-template>
-                      <span class="f8launcher-tags-label contribute" container="body" triggers="click"
+                      <span class="f8launcher-contribute-tag-label" container="body" triggers="click"
```

Comment:
Thank you for updating the selector names. However, we still need to edit the names in src/assets/stylesheets/shared/_labels.less. Without that change, these selectors do not exist -- the tags won't display properly.

I'm going to merge the PR as is and create a new PR to address the CSS issues separately.